ner/src/main/java/edu/illinois/cs/cogcomp/ner/config/NerBaseConfigurator.java

```diff
@@ -35,6 +35,10 @@
     public final static String PATH_TO_MODEL = "pathToModelFile";
     public final static String VIEW_NAME = "viewName";
 
+    public final static String TRAINED_ON = "all-data";
```

Comment:
To choose whether we use the models that are train on: 
- all data 
or 
- training data only

Reply:
Nice.